src/main/java/com/gluonhq/substrate/ProjectConfiguration.java

```diff
@@ -48,7 +48,7 @@
 
     private boolean usePrismSW = false;
     private boolean verbose = false;
-    private boolean usePrecompiledCode = true;
+    private boolean usePrecompiledCode;
```

Comment:
Probably we want to keep it true (via client plugin `main` is not called)

Reply:
Isn't [this](https://github.com/lazar-mitrovic/substrate/blob/additionalLibs/src/main/java/com/gluonhq/substrate/ProjectConfiguration.java#L72) equivalent (constructor should be always called, right)?

Reply:
Yes, you are right, but we do those settings in `SubstrateDispatcher`(via `main`, when running from command line, or gradle tests, or via client-plugin).

Reply:
This should be fixed now